feedparser/datetimes/rfc822.py

```diff
@@ -13,6 +13,12 @@
     'm': -12, 'y': 12,
     'met': 1, 'mest': 2,
 }
+daynames = ('mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun')
```

Comment:
When set() is called, the content is transformed (potentially) to make each element unique. A tuple does not do that.
Since daynames is supposed to be immutable, a tuple is more appropriated here.